### ISSUE TITLE
clouseau-spike-detection: Don't consider plugin crashes in the startup crashes detection

### ISSUE THREAD

**Issue by marco-c** [mozilla/clouseau#105]
As plugins crashes aren't really startup crashes.

**Comment by calixteman** [comment]
You're right, but I think that from release management point of view, the issue is the same: we risk to lose a lot of users because crashes occured to quickly.

**Comment by marco-c** [comment]
But they don't occur during startup, so they shouldn't be considered startup crashes. They should go in the email about normal crash spikes.

**Comment by marco-c** [comment]
For plugin crashes `uptime` means the amount of time elapsed since the plugin was instantiated, which might also be after one hour that Firefox has been running (depending on when you use that plugin).

**Comment by calixteman** [comment]
ok
